leaflet.photon.js

```diff
@@ -245,7 +245,14 @@ L.PhotonBaseSearch = L.PhotonBase.extend({
             detailsContainer = L.DomUtil.create('small', '', el),
             details = [],
             type = this.formatType(feature);
-        title.innerHTML = feature.properties.name;
+        if (feature.properties.name) {
+            title.innerHTML = feature.properties.name;
+        } else if (feature.properties.housenumber) {
+            title.innerHTML = feature.properties.housenumber;
+            if (feature.properties.street) {
+                title.innerHTML += ' ' + feature.properties.street;
```

Comment:
I never get to that point because AFAIK the housenumber/street order is not standard across the countries.
This was suppose to be an exercise left to the implementer, with a specific `formatResult`.
But we still can have a default behaviour more specific, people wanting to have control can still have their `formatResult`. 
What do you think?

Reply:
It would definitely help if the default implementation always returns something human-readable. Not even photon.komoot.de overwrites the formatter. So whatever happens here is likely to end up in the UI of most users.

I know that address formatting is tricky. I've made a more or less random choice here and would be happy to change it to anything else. (BTW Nominatim uses "<housenumber>, <street>"  which, I suspect, isn't a standard format anywhere in the world. So you might call that an equal opportunity choice. ;) ) A proper solution would probably check the locale of the user and format the address according to the preferences of the country but I felt that this was going too far for this simple plugin.